### ISSUE TITLE
change keywords

### ISSUE THREAD

**Issue by kmexter** [emo-bon/rocrate-metadata-generator-action#5]
The hard-coded keywords for the ro-crates for the observatories needs to be changed

Currently we have this:
"keywords":[
                "https://dwc.tdwg.org/terms/#materialsample",
                "https://schema.org/Dataset",
                "EMO BON",
                "https://vocabularyserver.com/asfa/index.php?tema=2185&/Biodiversity",
                "https://vocabularyserver.com/asfa/index.php?tema=2091&/benthic-communities",
                "https://vocabularyserver.com/asfa/index.php?tema=3505&/Long-term-changes",
                "https://vocabularyserver.com/asfa/index.php?tema=2146&/biology",
                "https://vocabularyserver.com/asfa/index.php?tema=1150&/marine-environment",
                "https://vocabularyserver.com/asfa/index.php?tema=2365&/genetics",
                "https://vocabularyserver.com/asfa/index.php?tema=6369&/sediment-samples",
                "https://vocabularyserver.com/asfa/index.php?tema=3248&/water-samples",
                "https://www.w3.org/TR/vocab-ssn/#SOSASampling",
                "https://rs.tdwg.org/terms/Event",
                "https://schema.org/Dataset"
            ],

what we need to have is this:

"keywords":["sample",
                "measurement",
                "biodiversity",
                "long-term-changes",
                "marine-environment",
                "genetics",
                "sediment-samples",
                "water-samples",
                "benthic-communities"
            ],

AND
"dcat:theme":[
                "http://semanticscience.org/resource/SIO_001050",
               "http://purl.obolibrary.org/obo/ExO_0000067",
                "https://vocabularyserver.com/asfa/?tema=2185",
                "https://vocabularyserver.com/asfa/?tema=3505",
                "https://vocabularyserver.com/asfa/?tema=1150",
                "https://vocabularyserver.com/asfa/?tema=2365",
                "https://vocabularyserver.com/asfa/?tema=6369",
                "https://vocabularyserver.com/asfa/?tema=3248",
                "https://vocabularyserver.com/asfa/?tema=2091",
            ],

This is for @laurianvm to include in her working-up of the loghsheet ro-crate example that we will pass on to ... @cedricdcc I think, who will modifiy the code that creates the ro-crates for the observatories.

**Comment by kmexter** [comment]
Once @marc-portier and @laurianvm have check that I understood this correctly, I can pass it on to cymon so he can add it to his MGF ro-crates

**Comment by kmexter** [comment]
And then @kmexter needs to make the changes to the ARMS ro-crates

**Comment by laurianvm** [comment]
included in the loghsheet ro-crate example ([see commit](https://github.com/emo-bon/observatory-bpns-crate/commit/78f26641e38b3ee8b23b04e0b011bd0ad4ff8e4b))

**Comment by laurianvm** [comment]
> Once @marc-portier and @laurianvm have check that I understood this correctly, I can pass it on to cymon so he can add it to his MGF ro-crates

dcat:theme should be likewise:
```
"dcat:theme":[
                {"@id":"http://semanticscience.org/resource/SIO_001050"},
                {"@id":"http://purl.obolibrary.org/obo/ExO_0000067"},
                {"@id":"https://vocabularyserver.com/asfa/?tema=2185"},
                {"@id":"https://vocabularyserver.com/asfa/?tema=3505"},
                {"@id":"https://vocabularyserver.com/asfa/?tema=1150"},
                {"@id":"https://vocabularyserver.com/asfa/?tema=2365"},
                {"@id":"https://vocabularyserver.com/asfa/?tema=6369"},
                {"@id":"https://vocabularyserver.com/asfa/?tema=3248"},
                {"@id":"https://vocabularyserver.com/asfa/?tema=2091"}
            ],
```

**Comment by kmexter** [comment]
I leave it up to you guys to tell me what I have to tell cymon to do, if it is more than changing the keyword paragraph and adding the theme paragraph. Can you copy here exactly the text to add/replace? 
And Laurian I need those same instructions so the can be done on the ARMS ro-crates also (but via the extra-metadata.json, not directly in the ro-crate file as that is re-written each time the repo is updated)

**Comment by laurianvm** [comment]
The instructions are normally already specified in [the google docs ](https://docs.google.com/document/d/1Uc2M_-tDBrQWJ5oO1FTAh4FEVc6jhh1NuS4PPzOBcOM/edit?tab=t.vxc7igwcuar8)

If easier, the extra-metadata.json in this repository can be used as an example with regards to keywords and dcat:theme;
I can also make the changes to the extra-metadata.json for ARMS if you like :)

**Comment by kmexter** [comment]
yes, I find the instructions in the googledocs very hard to follow
but yes, the extra-metadata file is a good example and yes please, do change the arms ones for me - the only difference is that they do not need the two keywords that are water-sample and sediment-sample

**Comment by laurianvm** [comment]
are those [extra_metadata.json](https://github.com/arms-mbon/data_workspace/blob/main/extra_metadata.json) and [ro-crate-metadata.json](https://github.com/arms-mbon/data_workspace/blob/main/ro-crate-metadata.json)?

**Comment by kmexter** [comment]
they are the extra_metdata.json files in the repos data_release_001, code_release_001, analysis_release_001, and data_workspace

**Comment by cymon** [comment]
Changed in [metagoflow-data-products-ro-crate](https://github.com/emo-bon/metaGOflow-data-products-RO-crate-example/blob/main/ro-crate-metadata.json-template). Done.

**Comment by kmexter** [comment]
do you want to add any keywords - something about annotations perhaps? or metagenomics? I invite you to consider that (but don't ask me to find them please)

**Comment by kmexter** [comment]
I will close this issue now. the mgf one has been done (tho I think we may be asking you, cymon, to add one extra bit that I had missed out of this issue) and the BPND example one is done.
